index.js

```diff
@@ -54,11 +56,15 @@ class Analytics {
       enumerable: true,
       value: typeof options.enable === 'boolean' ? options.enable : true
     })
-    axiosRetry(this.axiosInstance, {
-      retries: options.retryCount || 3,
-      retryCondition: this._isErrorRetryable,
-      retryDelay: axiosRetry.exponentialDelay
-    })
+    if (options.retryCount !== 0) {
+      axiosRetry(this.axiosInstance, {
+        retries: options.retryCount || 3,
+        retryDelay: axiosRetry.exponentialDelay,
+        ...(options.axiosRetryConfig || {}),
```

Comment:
```suggestion
        ...options.axiosRetryConfig,
```

Reply:
TIL 
I thought this safeguard was necessary. Turns out its not :P